docs/site/_includes/getting_started/existing/STEP_FINALIZE_RU.md

```diff
@@ -0,0 +1,89 @@
+<script type="text/javascript" src='{{ assets["getting-started.js"].digest_path }}'></script>
+<script type="text/javascript" src='{{ assets["getting-started-access.js"].digest_path }}'></script>
+
+Кластеры некоторых провайдеров могут требовать дополнительных действий, необходимых для запуска Deckhouse и его компонентов. Ниже приведены некоторые такие случаи. Если вы столкнулись с другими особенностями установки Deckhouse в существующем кластере, пожалуйста, опишите ваш опыт в [issue](https://github.com/deckhouse/deckhouse/issues). 
+
+{% offtopic title="Кластер в VK Cloud Solutions (MailRu Cloud Solutions)" %}
+<ul><li><p>В кластерах версии 1.21+ присутствует Gatekeeper (OPA), который требует выставления limits и requests подам. Pod Deckhouse архитектурно не имеет limits/requests, и, после установки Deckhouse, его Pod не будет запущен. При просмотре событий Deployment <code>deckhouse</code> вы можете увидеть ошибку:</p>
+<div class="highlight"><pre><code>admission webhook "validation.gatekeeper.sh" denied the request: [container-must-have-limits] container <...> has no resource limits...</code></pre></div>
+
+<p>Чтобы Deckhouse смог запуститься необходимо удалить Gatekeeper. Для удаления Gatekeeper выполните на узле, имеющем доступ к API кластера, следующие команды (потребуется установка <a href="https://helm.sh/" target="_blank">helm-клиента</a>):</p>
+{% snippetcut selector="gatekeeper-uninstall" %}
+```shell
+helm delete gatekeeper --namespace opa-gatekeeper
+kubectl delete crd -l gatekeeper.sh/system=yes
+```
+{% endsnippetcut %}
+</li>
+<li>Для версий кластера 1.20+ необходимо удалить также <code>metrics-server</code>. Для удаления <code>metrics-server</code> выполните на узле, имеющем доступ к API кластера, следующие команды (потребуется установка <a href="https://helm.sh/" target="_blank">helm-клиента</a>): 
+{% snippetcut selector="metrics-server" %}
+```shell
+helm -n kube-system uninstall metrics-server
+```
+{% endsnippetcut %}
+</li></ul>
+{% endofftopic %}
+
+<blockquote>
+<p>Некоторые компоненты Deckhouse по умолчанию не работают на master-узле. Если вам необходимо разрешить компонентам Deckhouse работать на master-узле, снимите с master-узла taint, выполнив следующую команду:</p>
+{% snippetcut %}
+```bash
+kubectl patch nodegroup master --type json -p '[{"op": "remove", "path": "/spec/nodeTemplate/taints"}]'
```

Comment:
There might be no nodegroup master. We need to explain how to deploy Deckhouse on any desired node